payload-examples/api.github.com/installation.created.payload.without-installation-prop.json

```diff
@@ -0,0 +1,32 @@
+{
+  "action": "created",
+  "repositories": [
+    {
+      "id": 186853002,
+      "node_id": "MDEwOlJlcG9zaXRvcnkxODY4NTMwMDI=",
+      "name": "Hello-World",
+      "full_name": "Codertocat/Hello-World",
+      "private": false
+    }
+  ],
+  "sender": {
+    "login": "Codertocat",
+    "id": 21031067,
+    "node_id": "MDQ6VXNlcjIxMDMxMDY3",
+    "avatar_url": "https://avatars1.githubusercontent.com/u/21031067?v=4",
+    "gravatar_id": "",
+    "url": "https://api.github.com/users/Codertocat",
+    "html_url": "https://github.com/Codertocat",
+    "followers_url": "https://api.github.com/users/Codertocat/followers",
+    "following_url": "https://api.github.com/users/Codertocat/following{/other_user}",
+    "gists_url": "https://api.github.com/users/Codertocat/gists{/gist_id}",
+    "starred_url": "https://api.github.com/users/Codertocat/starred{/owner}{/repo}",
+    "subscriptions_url": "https://api.github.com/users/Codertocat/subscriptions",
+    "organizations_url": "https://api.github.com/users/Codertocat/orgs",
+    "repos_url": "https://api.github.com/users/Codertocat/repos",
+    "events_url": "https://api.github.com/users/Codertocat/events{/privacy}",
+    "received_events_url": "https://api.github.com/users/Codertocat/received_events",
+    "type": "User",
+    "site_admin": false
+  }
+}
```

Comment:
The `installation` event has always an installation key

Reply:
I doubted about this one because, according to docs, it says
> The GitHub App installation. Webhook payloads contain the installation property when the event is configured for and sent to a GitHub App.

https://docs.github.com/en/free-pro-team@latest/developers/webhooks-and-events/webhook-events-and-payloads#installation

Reply:
In this case the `"installation"` object is the key part of the payload, it has many properties beyond just `id` and `node_id`. I'm sure that it's always present